src/ports/catalog/queries.ts

```diff
@@ -366,14 +370,47 @@ const getOwnersJoin = () => {
     .append('.nft as nfts GROUP BY nfts.item) AS nfts ON nfts.item = items.id')
 }
 
+const MAX_NUMERIC_NUMBER = '115792089237316195423570985008687907853269984665640564039457584007913129639935'
+
 const getMinPriceCase = (filters: CatalogQueryFilters) => {
+  /* 
+    This has a workaround to remove the NULLs from the LEAST function, because it will pick NULL as the lowest value.
+    To avoid this, we add the COALESCE with a very high number, so it will always pick the other value.
+  */
   return SQL`CASE
                 WHEN items.available > 0 AND items.search_is_store_minter = true 
-                `.append(filters.minPrice ? SQL`AND items.price >= ${filters.minPrice}` : SQL``)
-    .append(` THEN LEAST(items.price, nfts_with_orders.min_price) 
-                ELSE nfts_with_orders.min_price 
+                `
+    .append(filters.minPrice ? SQL`AND items.price >= ${filters.minPrice}` : SQL``)
+    .append(
```

Comment:
maybe we can do htis as a string and concat it so its easier to read? we shold be ok as we don't have any external value rright? I think its even taking it everything as a string now and not sql variables the max_numeric_number